### PR TITLE
Create boilerplate backend

### LINKED ISSUE #8: Set up boilerplate backend for the website
Set up a server and database and publish to Heroku. 

Potentially handle some devops/environment management but that can be separated out into a different issue

### PR DESCRIPTION
**Pull request by mahadzaryab1**
This pull request closes #8.

### CODE REVIEW

**Review by eyhhuang — COMMENTED** [review]
Left a few small comments but thanks for getting this started!! I'm looking into getting a Heroku account for Tech+, just needing to figure out the right pricing

**Comment by eyhhuang** [comment]
Btw, in the future, I don't think we need to link Portal project to Pull Requests. Since there was an associated issue for this task that was added to the Portal project, adding the PR to the portal project results in two project cards that essentially track the same task. So to reiterate, when creating issues, set the Project and assign yourself. When creating PRs, just link the issue in the PR description.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
-Also a portal to enable mentors/mentees to update their profiles
+## Running the Server-Side API
+The server-side API is a **dockerized** application, hence you will need the following installations on your machine: 
+- [Docker](https://docs.docker.com/desktop/)
+- [Docker-Compose](https://docs.docker.com/compose/)
+
+The API can then be run by running the following commands on your terminal: 
+```shell
+cd portal_api
+docker-compose build
```
[the remark this answers is not in the document]
Comment (mahadzaryab1) [reply README.md:14]: I looked up this issue and there's a lot of suggestions regarding an issue with the installation/restarting Docker. Do you run into this issue if you close Docker and restart it?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
 
-Main landing site for current/prospective members and sponsors to learn about Tech+
+## About
+The Tech+ UW Portal is the main landing site for current/prospective members and sponsors to learn about Tech+. Moreover, this portal also enables mentors and mentees to update their profiles and potentially maintain mentor-mentee relationships.
 
-Also a portal to enable mentors/mentees to update their profiles
+## Running the Server-Side API
+The server-side API is a **dockerized** application, hence you will need the following installations on your machine: 
+- [Docker](https://docs.docker.com/desktop/)
+- [Docker-Compose](https://docs.docker.com/compose/)
```
[the remark this answers is not in the document]
Comment (mahadzaryab1) [reply README.md]: Yeah - fixed!

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.vscode/
+__pycache__/
+venv/
```
[the remark this answers is not in the document]
Comment (mahadzaryab1) [reply .gitignore]: Yeah - fixed!

File: portal_api/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Use the Python3.7.2 container image
```
[the remark this answers is not in the document]
Comment (mahadzaryab1) [reply portal_api/Dockerfile]: My comment was incorrect - I'm using the latest version of Python. Fixed!

File: portal_api/.dockerignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+venv/
+__pycache__/
```
Comment (eyhhuang) [inline portal_api/.dockerignore:2]: can we add the newline for all the other files too? You can see it more easily on the Github UI which files still need a  newline!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eyhhuang) [reply README.md]: I just tried this again and no longer had the issue! Might have just been the case that the first time I tried it, Docker wasn't fully up and running.

**Review by eyhhuang — APPROVED** [review]
Small styling comment, but otherwise LGTM!

File: portal_api/.dockerignore (hunk 5 above)
Comment (mahadzaryab1) [reply portal_api/.dockerignore]: Done!

**Pull request merged**